app/models/perilune/trifle/stat.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Perilune
+  module Trifle
+    class Stat
```

Comment:
Same here, its singular.

Reply:
In the same time, dont think this needs to be separate class. It can be part of Perilune configuration (separate method that returns stats configuration based on passed in driver).